python/distrdf/test_friend_trees.py

```diff
@@ -107,6 +108,55 @@ def test_friend_tree_histo(self):
         os.remove("treeparent.root")
         os.remove("treefriend.root")
 
+    def test_friend_trees_7584(self):
```

Comment:
7584 does not really add much :smile: 
```suggestion
    def test_friend_trees(self):
```

Reply:
fixed in rebased commit